src/WebApi/Extensions/EndpointRouteBuilderExt.cs

```diff
@@ -34,4 +34,14 @@ public static class EndpointRouteBuilderExt
         .Produces(StatusCodes.Status204NoContent)
         .Produces(StatusCodes.Status404NotFound)
         .ProducesProblem(StatusCodes.Status500InternalServerError);
-}
+
+    public static RouteGroupBuilder MapApiGroup(this IEndpointRouteBuilder app, string groupName)
```

Comment:
Should we explicitly extends on WebApplication so this can only be called on root level?

Reply:
I like the suggestion.  Have updated the extension method.